admin/config/routes.rb

```diff
@@ -430,5 +430,7 @@
 
   resource :configuration, only: [:show, :update]
 
+  resources :shipping_skus, except: [:destroy]
```

Comment:
Why can't these be destroyed?

Reply:
they're destroyed through the bulk editing feature. there's also no `#destroy` action on the pricing skus controller.